model/src/pyrenew/model/rtinfectionsrenewalmodel.py

```diff
@@ -311,57 +313,41 @@ def sample(
 
         # Sampling initial infections
         I0, *_ = self.sample_I0(**kwargs)
-        I0_size = I0.size
         # Sampling from the latent process
-        latent_infections, *_ = self.sample_infections_latent(
+        post_seed_latent_infections, *_ = self.sample_infections_latent(
```

Comment:
Minor: Do you risk is low that someone misconstrues `seed` here for reproducibility / something involved in `numpyro.sample()` vs. interpreting `post_seed_latent_infections` as total infectiousness after seeded infections $I_{v:0}$ have been added?

I think the risk is low, given that those who might be interested in using this enough to peruse the source code would understand this.

Reply:
I think we should discuss this in a separate issue; have created one: #206

Reply:
> I think we should discuss this in a separate issue; have created one: #206

Thank you Dylan. I missed this.